### ISSUE TITLE
[BUG_SP] Working width detection error kockerling rebel classic 600t

### ISSUE THREAD

**Issue by Markus33-ITA** [Courseplay/Courseplay_FS22#450]
**Steps to reproduce**
I rented the Kockerling Rebel Classic 600t for tillage 39 and it set me the working width at 6.8m. Tried 3 times + 2 more (restarting the game) just before making the screens and it doesn't change

**Actual behavior**
incorrect working width detection

**Expected behavior**
correct detection of the working width

**Screenshots**
![fsScreen_2022_01_09_11_09_25](https://user-images.githubusercontent.com/96330304/148678092-f30b5fb6-e58b-4bb2-94ef-1de530b8266f.png)
![fsScreen_2022_01_09_11_09_45](https://user-images.githubusercontent.com/96330304/148678099-d115cb56-3788-4170-95d1-300636f40da5.png)
![fsScreen_2022_01_09_11_09_59](https://user-images.githubusercontent.com/96330304/148678118-d74eca37-f240-451c-8003-d81b20f8519c.png)

**Courseplay version**
Version used: 7.0.0.16

**Log**
[https://pastebin.com/M2FxVX6C](https://pastebin.com/M2FxVX6C)

**Map**
Map US

**Comment by Tensuko** [comment]
And when you generate a course with 6.8m and let it run. Do you have unworked areas ?

**Comment by Markus33-ITA** [comment]
[https://pastebin.com/KvW8KpcH](https://pastebin.com/KvW8KpcH)
I had to start again because I had set the measurement of 6m and I had already done half the work.
the job does it without problems. I think it is a Giants bug. he put 6m as work but looking more closely at the log, I see that there are 3.40 m to the right and left for a total of 6.8.
![fsScreen_2022_01_09_11_51_38](https://user-images.githubusercontent.com/96330304/148679407-5bb52c9f-8286-4194-ae1d-8e767541fe36.png)

I close the topic.
